### PR TITLE
Security Group Function

### PR DESCRIPTION
**Pull request by meongbego**

### CODE REVIEW

**Comment by anak10thn** [comment]
is done?

File: neo/libs/network.py [hunk 1]
```diff
@@ -24,3 +24,23 @@ def get_floatingips(session=None):
...
+
+
+def list_sec_group(session=None):
+    neutron = get_neutron_client(session)
+    sec_group =  neutron.list_security_groups()
+    return sec_group['security_groups']
+
+
+def rules_sec_groups(sec_group, session=None):
+    obj = list()
```
Comment (azzamsa) [inline neo/libs/network.py]: Please use appropriate name for obj.

File: neo/libs/network.py [hunk 2]
```diff
@@ -24,3 +24,23 @@ def get_floatingips(session=None):
...
+    neutron = get_neutron_client(session)
+    sec_group =  neutron.list_security_groups()
```
Comment (azzamsa) [inline neo/libs/network.py]: Can we use one liner for this ?

**Comment by azzamsa** [comment]
@meongbego thanks for the changes.

@anak10thn ready for final review.

**Comment by anak10thn** [comment]
@azzamsa approved?

**Comment by azzamsa** [comment]
@anak10thn sure :) yes.

**Pull request merged**